app/assets/javascripts/short-form/directives/checkbox-preference-item.html.slim

```diff
@@ -1,6 +1,6 @@
 div
   p.checkbox-block
-    input id="preferences-{{checkbox_pref_type}}" name="preferences" data-event="{{pref_data_event}}" ng-model="application.preferences[checkbox_pref_type]" type="checkbox" aria-labelledby="{{labelledby}}" ng-click='reset_preference_data(checkbox_pref_type)'
+    input id="preferences-{{checkbox_pref_type}}" name="preferences" data-event="{{pref_data_event}}" ng-model="application.preferences[checkbox_pref_type]" type="checkbox" aria-labelledby="{{labelledby}}" ng-click="reset_preference_data(checkbox_pref_type);refresh_member_dropdown()"
     label.checkbox-block_label for="preferences-{{checkbox_pref_type}}"
```

Comment:
yes I'm remembering this directive getting sort of beastly especially after the introduction of the "combined Live OR Work" preference. I'm wondering if this `ng-click` should be refactored to `ng-click="onPreferenceChange(checkbox_pref_type)"` and inside _that_ method it calls `reset_preference_data` and `refresh_member_dropdown`. 

Likewise the `refresh_member_dropdown` could be called within `select_liveWork_type` which is really the onSelect for that input (could be renamed if it needs to be clearer, I also realize we sometimes mix ruby + js naming conventions together, I am definitely at fault for a lot of it 😬 )